MAUI/DataForm/localization.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: Localization in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about Localization support in Syncfusion .NET MAUI DataForm(SfDataForm) control and more.
```

Comment:
Learn about the Localization support in Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.

Reply:
description changed